styles/components/avatars.css

```diff
@@ -1,7 +1,7 @@
 .avatar {
   border-radius: var(--border-radius-circle);
   display: inline-block;
-  margin: 0 0.2rem;
+  margin: 0.1rem;
```

Comment:
why avatars change in button component

Reply:
Yeah, will keep it in mind from next PR.